core/lib/dal/src/transactions_web3_dal.rs

```diff
@@ -550,6 +573,19 @@ mod tests {
             .get_transaction_by_hash(H256::zero(), L2ChainId::from(270))
             .await;
         assert!(web3_tx.unwrap().is_none());
+
+        let execution_info = conn
+            .transactions_web3_dal()
+            .get_unstable_transaction_execution_info(tx_hash)
+            .await
+            .unwrap()
+            .unwrap();
```

Comment:
I guess this `unwrap` should be `.expect("Transaction execution info is missing in the DAL")` (normally we `unwrap` only in the places that are assumed to always hold, while here it's a part of the test).